tools/run_tests/xds_k8s_test_driver/tests/outlier_detection_test.py

```diff
@@ -29,11 +29,21 @@
 _XdsTestServer = xds_k8s_testcase.XdsTestServer
 _XdsTestClient = xds_k8s_testcase.XdsTestClient
 
+# Testing consts
+_QPS = 100
+_REPLICA_COUNT = 5
 
 class OutlierDetectionTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
 
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
+        if config.client_lang == _Lang.CPP:
+            return not config.version_lt('v1.48.x')
```

Comment:
`version_lt` is no longer available, please use `version_gte`

Reply:
Fixed.